package.json

```diff
@@ -50,7 +50,8 @@
     "elliptic@6.5.4": "^6.5.7",
     "fast-xml-parser@^4.3.4": "^4.4.1",
     "send": "^0.19.0",
-    "ws@7.4.6": "^7.5.10"
+    "ws@7.4.6": "^7.5.10",
+    "color-string": "^1.5.5"
```

Comment:
Fixing vulnerability with `color-string`

https://github.com/advisories/GHSA-257v-vj4p-3w2h

<img width="1211" alt="Screenshot 2025-03-18 at 3 43 43 PM" src="https://github.com/user-attachments/assets/bc84b970-58c7-40dc-a23c-f45b624d5725" />
<img width="908" alt="Screenshot 2025-03-18 at 3 44 00 PM" src="https://github.com/user-attachments/assets/0833a631-8990-4e71-a96e-cac35a2b53a4" />

Reply:
Looks like we can't fix this vulnerability without breaking the jazzicons package we may need to update it on the library side